components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/AuthenticationEndpointUtil.java

```diff
@@ -323,6 +323,8 @@ public static String getErrorCodeToi18nMapping(String errorCode, String subError
                 return Constants.ErrorToi18nMappingConstants.INVALID_CLIENT_IN_TENANT_I18N_KEY;
             case Constants.ErrorToi18nMappingConstants.CLIENT_NOT_AUTHORIZED_TO_USE_REQUESTED_GRANT_TYPE:
                 return Constants.ErrorToi18nMappingConstants.CLIENT_NOT_AUTHORIZED_TO_USE_REQUESTED_GRANT_TYPE_I18N_KEY;
+            case Constants.ErrorToi18nMappingConstants.AUTHENTICATION_FLOW_TIMEOUT_AUTHENTICATION_FLOW_TIMEOUT_DESCRIPTION:
```

Comment:
any specific reason to have the constant named as `AUTHENTICATION_FLOW_TIMEOUT_AUTHENTICATION_FLOW_TIMEOUT_DESCRIPTION` ?
can't we use `AUTHENTICATION_FLOW_TIMEOUT` and `AUTHENTICATION_FLOW_TIMEOUT_I18N_KEY`?

Reply:
I used the existing pattern we have used previously with other i18n codes. It maybe to give indication that the key is the combination of error status and status msg.